release-post.md

```diff
@@ -0,0 +1,68 @@
+# Universal Sierra Compiler
+
+The Software Mansion Team has created a new tool, Universal Sierra Compiler (further abbreviated as USC).
+
+USC is the tool for Sierra compilation - it compiles any ever-existing Sierra version to CASM.
+
+The idea behind USC is simple, yet it might make things easier for some tools.
+It bundles all Sierra compilers previously released, and depending on the version contained inside the Sierra JSON, it chooses the right compiler.
+Therefore it allows any tool to be independent of Sierra version.
+
+Universal Sierra Compiler might become especially useful for nodes, which have to interact with contracts deployed in different versions of Sierra, but also any other tool that depends on the versioning of Sierra.
+
+
+## Usage
+
+| Command          | Purpose                               | Input                        | Input as JSON of Cairo struct                              | Output as JSON of Cairo struct |
+|------------------|---------------------------------------|------------------------------|------------------------------------------------------------|-|
+| compile-contract | compile Sierra of a Starknet contract | output of `cairo-compile`    | cairo_lang_starknet_classes::contract_class::ContractClass | cairo_lang_starknet::casm_contract_class::CasmContractClass |
+| compile-raw      | compile Sierra of a Cairo program     | output of `starknet-compile` | cairo_lang_sierra::program::Program                        | |
+|                  |                                       |                              |                                                            | |
+
+
+```shell
+$ universal-sierra-compiler compile-raw --sierra-path ./path/to/sierra.json
+
+{"assembled_cairo_program": ...}
+```
+
+
+## Installation
+
+Use the provided script to install the latest version of Universal Sierra Compiler:
+
+```shell
+curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh
+
+...
+universal-sierra-compiler (v1.0.0) has been installed successfully.
+```
+
+It is also possible to install a specific version of USC by passing, for example:
+
+```shell
+| sh -s v2.0.0-rc0
+```
+
+however, since USC aims to be backwards-compatible itself, it should be enough to always use the latest version (e.g. on CI).
+
+Note that versions of USC that are `release candidate` (rc) will not be fetched when installing the latest version. For that you would need to specify a concrete version.
```

Comment:
> For that you would need to specify a concrete version

I think it'd be useful to describe when i'd like to do that (when i want to adjust/test for new sierra)

Reply:
so the problem i have is that as a tool that uses USC, i'd still have to bump it when i'd want to use a new sierra version from cairo `rc`

i guess it wouldn't make much sense to do a normal release once new sierra comes out in cairo `rc`, since it can change in the next `rc` version

but maybe we could have an option to download the latest rc release? (unless its already possible somehow)

Reply:
> i guess it wouldn't make much sense to do a normal release once new sierra comes out in cairo rc, since it can change in the next rc version

unless the change is annotated with patch in sierra version? (e.g. `rc0` -> 1.5.0, `rc1` -> 1.5.1)